### DISCUSSION TITLE
Microsoft Dynamics CRM

### DISCUSSION THREAD

**Discussion by daniellelecomte** [solspace/craft-freeform#202] [Feature Requests]
Any plans to add this to the already amazing list of CRM integrations?

**Comment by kjmartens** [comment]
From what I understand, this is an extremely complex integration to add support for, and each setup can be very opinionated as well (so possibly harder to make a 1 size fits most solution).

**Reply by daniellelecomte** [reply]
Appreciate the reply and insight! Have you seen much success with the POST Forwarding option here?

**Reply by kjmartens** [reply]
That could be an option. I think I have heard of one other customer that used POST Forwarding along with some other third party (I think) service to communicate between Freeform and MS Dynamics, but I cannot recall what that service was called.

**Comment by TheFunkyMonk** [comment]
We have an upcoming client using Dynamics and would love to be able to recommend Freeform to them if this is possible.

**Reply by kjmartens** [reply]
Thanks for checking in @TheFunkyMonk. Now that Freeform 5 is out, I will discuss this with the team and see if it's something we can add support for in the near future. 🙂

**Comment by Coysh** [comment]
Adding my +1 for this. I've got experience with the MS Dynamics API, but it seems like Solspace do themselves [https://solspace.com/case-studies/microsoft-dynamics-to-craft-cms-integration]

Also worth noting that Formie does have a MS Dynamics integration [https://verbb.io/craft-plugins/formie/docs/integrations/crm]

**Comment by croxton** [comment]
Just wanted to bump this FR as we have an upcoming job that requires this integration.

**Reply by Coysh** [reply]
Hi Mark. If it helps I've done some implementation with FreeForm using the API. Could happily help you out a little? Feel free to email me at tim@coysh.digital

**Comment by kjmartens** [comment]
The next version of Freeform will include a beta integration for MS Dynamics 365 🙂

**Comment by kjmartens** [comment]
This has now been added in Freeform 5.16.0 🙂
